### PR TITLE
TSDataset user guide revision

### PR DESCRIPTION
**Pull request by TheaperDeng**
I include some detailed information in this version of TSDataset UG.

You can preview the doc here: https://junweitestdoc.readthedocs.io/en/tsdataset-user-guide/doc/Chronos/Overview/chronos.html#data-processing-and-feature-engineering

#### What will be included in the UG
1. What can we do through TSDataset ?
2. How can we do these things through TSDataset ?
3. (not for every subsection) Why should we do this?

#### What will NOT be included in the UG
1. Overview block diagram (future overview)
2. Detailed behavior of each parameter (API doc)
3. Runnable code (Quick start)

### CODE REVIEW

**Comment by TheaperDeng** [comment]
@cabuliwallah @liangs6212 pls raise any issue/confusion/typo in this UG.

File: docs/readthedocs/source/doc/Chronos/Overview/chronos.md [hunk 1]
```diff
@@ -197,8 +197,82 @@ DBScanDetector uses DBSCAN clustering algortihm for anomaly detection.
...
+
+If you are building a prototype for your forecasting/anomaly detection task and you need to split you dataset to train/valid/test set, you can use `with_split` parameter.[`TSDataset`](../../PythonAPI/Chronos/tsdataset.html) supports split with ratio by `val_ratio` and `test_ratio`.
+#### **6.3 Time series dataset preprocessing**
+[`TSDataset`](../../PythonAPI/Chronos/tsdataset.html) now supports [`impute`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.impute), [`deduplicate`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.deduplicate) and [`resample`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.resample). You may fill the missing point by [`impute`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.impute) in differnt modes. You may remove the records that are totally the same by [`deduplicate`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.deduplicate). You may change the sample frequency by [`resample`](../../PythonAPI/Chronos/tsdataset.html#zoo.chronos.data.tsdataset.TSDataset.resample). A typical cascade call for preprocesing is:
```
Comment (yushan111) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos.md:239]: different

File: docs/readthedocs/source/doc/Chronos/Overview/chronos.md [hunk 2]
```diff
@@ -197,8 +197,82 @@ DBScanDetector uses DBSCAN clustering algortihm for anomaly detection.
...
+
+#### **6.1 Basic concepts**
+A time series can be interpreted as a sequence of real value whose order is timestamp. While a time series dataset can be a combination of one or a huge amount of time series, since users may collect different time series in the same/different period of time (e.g. An AIops dataset may have CPU usage ratio and memory usage ratio data for two servers at a period of time. This dataset contains four time series). 
+
+In [`TSDataset`](../../PythonAPI/Chronos/tsdataset.html), we provides **2** possible dimensions to construct a high dimension time series dataset (i.e. **feature dimension** and **id dimension**).
+* feature dimension: Time series along this dimension might be independent or related still they are assumed to have **different patterns and distributions** and collected on the **same period of time**. For example, the CPU usage ratio and Memory usage ratio for the same server at a period of time.
+* id dimension: Time series along this dimension are assumed to have the **same patterns and distributions** and might by collected on the **same or differnt period of time**. For example, the CPU usage ratio for two servers at a period of time.
```
Comment (yushan111) [inline docs/readthedocs/source/doc/Chronos/Overview/chronos.md:211]: different

**Comment by cabuliwallah** [comment]
potential errors:

- directedly -> directly
- ...used in AutoTSEstimator or be converted to a pandas... ->...used in AutoTSEstimator. It can also be converted to a pandas...
- while provided -> while providing
- we provides ->we provide
- from a pandas dataframe though -> rom a pandas dataframe through
-  identifier that distinguish ->  identifier that distinguishes
- cascade call for preprocesing -> cascade call for preprocessing
- Detailed behavior please refer to the API doc `roll` -> Please refer to the API doc `roll` for detailed behavior
- A typical calling -> A typical call

**Comment by TheaperDeng** [comment]
> potential errors:
> 
> * directedly -> directly
> * ...used in AutoTSEstimator or be converted to a pandas... ->...used in AutoTSEstimator. It can also be converted to a pandas...
> * while provided -> while providing
> * we provides ->we provide
> * from a pandas dataframe though -> from a pandas dataframe through
> * identifier that distinguish ->  identifier that distinguishes
> * cascade call for preprocesing -> cascade call for preprocessing
> * Detailed behavior please refer to the API doc `roll` -> Please refer to the API doc `roll` for detailed behavior
> * A typical calling -> A typical call
> * While a time series dataset can be a combination of one or a huge amount of time series, since users may collect different time series in the same/different period of time. ->
>   While a time series dataset can be a combination of one or a huge amount of time series. It may contain multiple time series since users may collect different time series in the same/different period of time.
> * Time series along this dimension might be independent or related still they are assumed to have different patterns and distributions and collected on the same period of time. ->
>   Time series along this dimension might be independent or related. Though they may be related, they are assumed to have different patterns and distributions and collected on the same period of time.

updated

**Comment by TheaperDeng** [comment]
pr validation will be run after merging

**Pull request merged**